libtbx/auto_build/bootstrap.py

```diff
@@ -2024,8 +2031,8 @@ def rebuild_docs(self):
     pass
 
 class DIALSBuilder(CCIBuilder):
-  CODEBASES_EXTRA = ['dials', 'xia2']
-  LIBTBX_EXTRA = ['dials', 'xia2', 'prime', 'iota', '--skip_phenix_dispatchers']
+  CODEBASES_EXTRA = ['dials', 'dxtbx', 'xia2']
```

Comment:
The CCI builder configures dxtbx so it also needs dxtbx as a codebase.  cctbxlite is the builder explicitly designed without dxtbx.  Using the cctbx builder currently fails with:

`RuntimeError: Module not found: dxtbx`

When dxtbx is added as a code base to the CCI builder, adding it explicitly to the DIALS, XFEL, and Phenix builders won't be necessary.  It will also then be picked up by the Labelit builder, where it is also currently missing.